src/Adapter/Feature/CommandHandler/BulkDeleteFeatureHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Feature\CommandHandler;
+
+use PrestaShop\PrestaShop\Adapter\Feature\Repository\FeatureRepository;
+use PrestaShop\PrestaShop\Core\Domain\Feature\Command\BulkDeleteFeatureCommand;
+use PrestaShop\PrestaShop\Core\Domain\Feature\CommandHandler\BulkDeleteFeatureHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\BulkFeatureException;
+use PrestaShop\PrestaShop\Core\Domain\Feature\ValueObject\FeatureId;
+use PrestaShop\PrestaShop\Core\Exception\BulkActionExceptionInterface;
+use PrestaShop\PrestaShop\Core\Util\AbstractBulkActionHandler;
+
+class BulkDeleteFeatureHandler extends AbstractBulkActionHandler implements BulkDeleteFeatureHandlerInterface
+{
+    /**
+     * @var FeatureRepository
+     */
+    private $featureRepository;
+
+    public function __construct(
+        FeatureRepository $featureRepository
+    ) {
+        $this->featureRepository = $featureRepository;
+    }
+
+    public function handle(BulkDeleteFeatureCommand $command): void
+    {
+        $this->handleBulkAction(array_map(static function (FeatureId $featureId): int {
```

Comment:
Following the comment I made on the abstract, I'm not sure it's relevant to perform this transformation into integers and back to VO again just to have some stricter typing in the abstract methods